website/views.py

```diff
@@ -144,6 +147,16 @@ def index():
 
     user_id = get_current_user_id()
     if user_id:  # Logged in: return either landing page or user home page
+        if waffle.switch_is_active('ember_dashboard'):
```

Comment:
Isn't the flag for the dashboard `ember_home_page`? Also will this be reached, given `@ember_flag_is_active`?

Reply:
Weird. I bet I have an outdated develop, since I did not add this line in this PR. Will investigate